conda_lock/src_parser/__init__.py

```diff
@@ -101,6 +105,66 @@ def validate_hash(cls, v: HashModel, values: Dict[str, typing.Any]) -> HashModel
         return v
 
 
+class TimeMeta(StrictModel):
+    """Stores information about when the lockfile was generated."""
+
+    created_at: str = Field(..., description="Time stamp of lock-file creation time")
+
+    @classmethod
+    def create(cls) -> "TimeMeta":
+        import time
+
+        return cls(created_at=datetime.datetime.utcnow().isoformat() + 'Z')
```

Comment:
```suggestion
        return cls(created_at=datetime.datetime.utcnow().isoformat(timespec="seconds") + 'Z')
```

By default Python's datetime includes microseconds and that may not be useful to have a "portable" ISO format, plus the microseconds probably don't matter that much for our usecase. The [`timespec` parameter](https://docs.python.org/3/library/datetime.html#datetime.datetime.isoformat) was added in Python 3.6 to handle cases like that.